packages/trader/src/Modules/CFD/Containers/cfd-password-manager-modal.tsx

```diff
@@ -155,12 +170,13 @@ const InvestorPasswordManager = ({
     setPasswordType,
     toggleModal,
     validatePassword,
-}) => {
+}: TInvestorPasswordManager) => {
     if (is_submit_success_investor) {
         return <CFDPasswordSuccessMessage toggleModal={toggleModal} is_investor />;
     }
 
-    const initial_values = { old_password: '', new_password: '', password_type: 'investor' };
+    const type_investor = 'investor';
```

Comment:
What is the reason to introduce the variable? Would you like to use it at line 256 (old line 240)?

Reply:
I believe this variable is not needed since we only use it in one place!

Reply:
Hard-coding something that has "password" keyword will fail sonarcloud tests and that's why I saved the value that we need in a variable and used the variable for password_type instead of hard-coding it to pass the tests.

Reply:
@hirad-rewok in this case, I suggest using this variable here as well at line 256 (old line 240):
```
<Button
                                className='cfd-password-manager--button'
                                type='button'
                                onClick={() => {
                                    setPasswordType(**type_investor**);
                                    multi_step_ref.current?.goNextStep();
                                }}
                                text={localize('Create or reset investor password')}
                                tertiary
                                large
                            />
```
what do you think? 😊